### ISSUE TITLE
Make the lessons more modular

### ISSUE THREAD

**Issue by abbycabs** [mozilla/open-leadership-training-series#45]
I'm getting a lot of feedback that more modular lessons would be helpful since projects are at different stages / different domains. Let's go through and remove language requiring each module to be done sequentially.

**Comment by betatim** [comment]
Is it really the sentence at the start of a section telling you "you should know X before doing Y" that puts people off?

I sometimes find that I want to dive into one specific thing but it takes me ages to find it. So for me having a big/expanded table of contents (basically like the sidebar but all unfolded) would be helpful for quickly finding stuff.

**Comment by abbycabs** [comment]
good point! I'll have to think about this more. I'll keep collecting feedback and I go through all the debriefs!

Would love to hear feedback from others on here!

**Comment by betatim** [comment]
Maybe just tuning the language a bit from "you must read X before Y" to "it is probably a good idea to read X (or have a basic idea about it) before reading Y" would help counter the notion that it is all or nothing.

**Comment by ghost** [comment]
Would the word "prerequisite(s)" be a put off?  If not, perhaps listing them could create the modular feeling that is needed.

**Comment by betatim** [comment]
A thing I miss is a boring old "Contents" page with all the chapters and sub-chapters visible. It would help me quickly finding things back that I know exist somewhere. Maybe we could add one after the glossary. Might require some advanced gh-pages foo to auto-generate

**Comment by abbycabs** [comment]
Auto-generating that is possible! 
This how I generated contents at the beginning of each section:
https://github.com/mozilla/open-leadership-training-series/blob/gh-pages/_layouts/page.html#L51

**Comment by zee-moz** [comment]
Hey @acabunoc @betatim I TOTALLY get how it feels very rigid and structured. And I agree stand-alone, mix and match modules would be more useful. 

(Some background: when Chad and I first started working on this, we wanted to make sure that the materials were comprehensive, so people could start with an idea and get all the way to sustained projects, and have everything they needed to do each step properly... so you're not jumping into opening your project on GitHub before you have a good project README and roadmap to put up, and you're not planning an event before you have a COC or contributor guidelines or even a clear way of using contributor guidelines.)

I agree we should take out the "you should have done x x and x" but probably still need to make notes where the exercises call for materials that would have been created in previous sections (like adding a good README as a first file to your GH repo). And also call out things that you could go without but probably shouldn't, like "before you jump into organizing an event, be sure you have a COC" and link to that section.

I also wonder if an "open project checklist" might be useful-- maybe up front, to structure the whole experience? Would this checklist be a good tool to put in the README, to help people think about what products they need and where learn to make them, if they're not working through the sequence?

**Comment by ghost** [comment]
+1 on the checklist idea, but I would rather link it to the README.md so the ones who want to use it can simply download that checklist file instead the README.md.

**Comment by betatim** [comment]
Started browsing the OLTS Jr. doc link in https://github.com/mozilla/open-leadership-training-series/issues/66 and thought: hey maybe a narrated overview (like OLTS Jr.??) of what is where and how it fits together/what builds on what would help people.

For me OLTS is meant to be like a good (technical) book: you read it cover to back the first, second and third times. Then you come back later to use it as a place to look up stuff: "Want to create a X, it is explained here somewhere, but where...?" this part is what I find a bit tricky, everything expanded TOC or a "heavy weight TOC" or a text with lots of links that you can scan quickly would be a good idea I think.

---

Can you make a small example of what the checklist would look like? Can't quite picture it right now :(
